csv2bufr/cli.py

```diff
@@ -110,12 +118,40 @@ def transform(ctx, csv_file, mapping, output_dir, station_metadata, verbosity):
         except Exception as err:
             raise click.ClickException(err)
 
+    # load JSON template
+    if template is not None:
+        if not os.path.isfile(template):
+            template_file = f"{MAPPINGS}{os.sep}{template}.json"
+        else:
+            template_file = template
+        try:
+            with open(template_file) as fh:
+                template = json.load(fh)
+        except Exception as err:
+            raise click.ClickException(err)
+
     click.echo("Writing data to file")
     for item in result:
         filename = f"{output_dir}{os.sep}{item}.bufr4"
         with open(filename, "wb") as fh:
             fh.write(result[item].read())
 
+        # convert to JSON, ideally we would do this from in memory object
+        # but I can't figure out how to do this with eccodes.
+        fh = open(filename, "rb")
+        handle = codes_bufr_new_from_file(fh)
+        fh.close()
+
+        result[item].seek(0)
+        if template is not None:
+            json_dict = bufr_to_json(handle, template)
+            json_dict["md5sum"] = item
+            json_dict["bufr_string_b64"] = \
+                base64.b64encode(result[item].read()).decode("utf-8")
+            filename = f"{output_dir}{os.sep}{item}.json"
+            with open(filename, "w") as fh:
```

Comment:
Let's keep the inline BUFR out of the JSON proper?  We can figure this out better by injecting a link relation with `"rel": "alternate"` to dereference the BUFR from it's online location.